odoo/addons/base/models/res_partner.py

```diff
@@ -234,11 +234,13 @@ def default_get(self, default_fields):
         readonly=False, store=True,
         help='The internal user in charge of this contact.')
     vat = fields.Char(string='Tax ID', index=True, help="The Tax Identification Number. Values here will be validated based on the country format. You can use '/' to indicate that the partner is not subject to tax.")
+    vat_value = fields.Char(compute='_compute_vat_value', readonly=True)
```

Comment:
Why not use directly `vat` and `company_registry`. I don't really understand this PR.

Reply:
Because of the several xpath on vat and company registry, actually it's explain in the commit message :sweat_smile: 
I asked him to remove it and we'll use something else.